README.md

```diff
@@ -0,0 +1,36 @@
+# Welcome to ppm-telegram-bot 👋
+![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)
+
+> Integration with telegram via bot platform. Uses for internal meetup needs
+
+## Install
+
+We use `poetry` as dependency manager, so it's the fastest way to install all stuff:
+
+```sh
+poetry install
+```
+
+## Usage
+In general, app is just FastAPI-based web-app with aiogram for interaction with telegram. So, for running it, you can
+ use any ASGI server, e.g. uvicorn:
```

Comment:
```suggestion
In general, ppm-telegram-bot is just a FastAPI web app with aiogram to interact with Telegram.

To run it you can use any ASGI server like [uvicorn](https://www.uvicorn.org/),
[gunicorn with uvicorn workers](https://www.uvicorn.org/deployment/#gunicorn)
or [daphne](https://github.com/django/daphne/#running), e.g.:
```

Reply:
Why? I add uvicorn just as an example of ASGI server and don't want to list all ASGI servers 🤔

Reply:
Well, actually, yeah, it might not be a good idea to _list_ all of them. But then, I'd rather cover the gunicorn with uvicorn running, instead, as a more production-suitable solution.

Reply:
LGTM